content/applications/sales/sales/send_quotations/get_paid_to_validate.rst

```diff
@@ -1,53 +1,93 @@
-and saves lots of time in a basic sales process.
+The Odoo *Sales* application provides customers with the ability to confirm orders, via an online
+payment, directly on a sales order. Once the sales order is electronically paid for by the customer,
+the salesperson attached to the sales order is instantly notified that the order is confirmed.
 
-Enable online payment
-=====================
+Activate online payments
+========================
 
-Go to :menuselection:`Sales --> Configuration --> Settings` and activate the **Online Payment**
-feature.
+In order to have customers confirm orders with an online payment, the *Online Payment* setting
+**must** be activated.
 
-.. image:: get_paid_to_validate/get_paid_1.png
-   :align: center
-   :class: img-thumbnail
-   :alt: How to enable online payment on Odoo Sales?
+To activate the *Online Payment* feature, go to :menuselection:`Sales app --> Configuration -->
+Settings`, scroll to the :guilabel:`Quotations & Orders` heading, and activate the :guilabel:`Online
+Payment` feature by checking the box next to it, and clicking :guilabel:`Save`.
 
-There, you will have direct access to the **Payment Providers** page. It will allow you to select
-and configure your providers of choice. Before creating or modifying a payment provider, be sure to
-check out our documentation about how to be paid with payment providers such as
-:doc:`/applications/finance/payment_providers/paypal`,
-:doc:`/applications/finance/payment_providers/authorize`, and others in the
-:doc:`/applications/finance/payment_providers` documentation.
+.. image:: get_paid_to_validate/online-payment-setting.png
+   :align: center
+   :alt: The online payment setting in the Odoo Sales application.
 
 .. note::
-   If you are familiar with this documentation: :doc:`quote_template`; you
-   can activate or not the **Online Payment** feature for each template you use, under their
-   confirmation tab.
+  To enable this feature on a standard quotation, click the checkbox for the :guilabel:`Payment`
+  feature option, located in the :guilabel:`Online confirmation` field, on the :guilabel:`Other
+  Info` tab.
+
+   .. image:: get_paid_to_validate/online-payment-option-quotation.png
+      :align: center
+      :alt: The online payment setting on a standard quotation in Odoo Sales.
+
+   To enable this feature on a quotation template, click the checkbox for the :guilabel:`Payment`
+   feature option, located in the :guilabel:`Online confirmation` field of the quotation template
+   form.
+
+   .. image:: get_paid_to_validate/online-payment-option-quotation-template.png
+      :align: center
+      :alt: The online payment setting on quotation template forms in Odoo Sales.
+
+Beneath the :guilabel:`Online Payment` option on the *Sales* :guilabel:`Settings` page, there's a
+:guilabel:`Default Quotation Validity` field. In this field, there's the option to add a specific
+number of days for quotations to remain valid by default.
```

Comment:
I think this section can be moved up to immediately follow the instruction to check the "Online Payment" feature box, so people don't need to jump back to the Sales > configuration > settings page after going to the quotation